docs/management/admin/response-actions-config.asciidoc

```diff
@@ -25,66 +24,107 @@ preview::[]
 
 * <<endpoint-management-req,{elastic-sec} feature privileges>>: **All** for the response action features, such as **Host Isolation**, that you want to perform.
 
-* Endpoints must have actively running SentinelOne agents installed.
+* Endpoints must have actively running third-party agents installed.
 --
 
-Configuration requires the following general steps. Expand the steps and follow the links for detailed instructions:
+Expand a section below for your endpoint security system:
 
-. **Generate API access tokens in SentinelOne.** You'll need these tokens in later steps, and they allow {elastic-sec} to collect data and perform actions in SentinelOne. 
+.**Set up CrowdStrike response actions**
+[%collapsible]
+====
+// NOTE TO CONTRIBUTORS: These sections have very similar content. If you change anything 
+// in this section, apply the change to the other sections, too.
+
+. **Create an API client in CrowdStrike.** Refer to CrowdStrike's docs for instructions on creating an API client.
++
+- Give the API client the minimum privilege required to read CrowdStrike data and perform actions on enrolled hosts.
```

Comment:
Lets double check this step is correct, because this currently states that the 1 API key with full privileges to read and also execute response actions. This provides the integration policy with a token that IMO has too many unnecessary privileges.

Reply:
Makes sense. The demo video and other materials I was able to find for CrowdStrike didn't specify multiple API clients, but we recommend 2 API keys for SentinelOne and could do something similar for CrowdStrike.

Reply:
Unfortunately I cannot confirm this - but you both may be right. Ingesting data might use a different clientId/secret than response actions - but it's rather Crowdstrike specific that I do not have information of ;/

Reply:
OK, I can try to reword this to be a little less prescriptive about exact number of API clients, give more general guidelines and leave it up to the user's discretion.